frontend/components/commons/sidebar/SidebarButton.vue

```diff
@@ -36,13 +35,23 @@ export default {
       required: true,
     },
     type: {
```

Comment:
It's better to not used keyword from Vuejs/js => rename the prop 'type' in something else